src/components/home/_home.scss

```diff
@@ -0,0 +1,49 @@
+.home-main {
+  figure {
+    width: 60%;
+    margin: 0 auto;
+  }
+
+  &--body {
+    width: 90%;
+    margin: 4rem auto;
+
+    .hero-text {
+      font-size: 1.4rem;
+    }
+  }
+}
+
+@media screen and (min-width: 769px) {
+  .home-main {
+    margin-left: 257px;
```

Comment:
you may consider keeping this property in relative units

Reply:
The absolute unit is used for the exactness with the width of the sidenavigation